DNN Platform/Website/admin/Skins/Logo.ascx.cs

```diff
@@ -58,6 +64,40 @@ protected override void OnLoad(EventArgs e)
                         {
                             this.imgLogo.ImageUrl = imageUrl;
                             logoVisible = true;
+                        }
+
+                        if (this.InjectSVG.GetValueOrDefault() == true)
+                        {
+                            logoVisible = false; // hide the <img>, we are going to inject an <svg> tag instead.

```

Comment:
This isn't necessary as the logo is already not visible per the declaration above

Reply:
actually, it will be set to visible in the logic block above.  I was trying to not change the existing change as much as possible.

Reply:
In all reality that means that we might want to change this process to be even further up the chain as there is no need to do all of that extra work on every load if we are doing it the SVG way